themes/grav/scss/preset.scss

```diff
@@ -1080,14 +1080,14 @@ form {
         transition: background-color 0.5s ease;
     }
 
-    &.switch-toggle input.highlight:checked + label {
```

Comment:
I may have confused the two; white on blue should then be from highlighting, and white on grey from checked? The main problem is this, where `input.highlight:checked + label` needs to operate without `:checked`.

Reply:
its a radio tho, not a checkbox.. something is always supposed to be checked.

Reply:
If the input-field is required, yes, but that's not necessarily the case. Especially in blueprints, if I default one value to be checked, that for example has the side-effect of being saved to the FrontMatter of a Page. This inadvertently makes explicit what is already implicit from the extension's options, and so to avoid polluting the resulting file I would want to highlight the default rather than check it.